p2p/host/relay/autorelay.go

```diff
@@ -247,7 +247,15 @@ func (ar *AutoRelay) discoverRelays(ctx context.Context) ([]peer.AddrInfo, error
 
 	ctx, cancel := context.WithTimeout(ctx, 30*time.Second)
 	defer cancel()
-	return discovery.FindPeers(ctx, ar.discover, RelayRendezvous, discovery.Limit(1000))
+	var ret []peer.AddrInfo
+	ch, err := ar.discover.FindPeers(ctx, RelayRendezvous, discovery.Limit(1000))
```

Comment:
This should be equivilent to the FindPeers util function in go-libp2p-discovery, to convert the channel into a slice.